bus-arrival/index.html

```diff
@@ -2,13 +2,16 @@
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 <title>Bus arrival times</title>
+<link rel="apple-touch-icon" href="../icons/icon-192.png">
```

Comment:
Can also move this `link` tag below all the other `link` tags since it's less important :)